crates/common/tedge_utils/Cargo.toml

```diff
@@ -11,20 +11,27 @@ description = "tedge_utils provide utilities for thin-edge.io components"
 # No features on by default
 default = []
 logging = ["tracing", "tracing-subscriber"]
+fs-notify = ["inotify", "async-stream", "maplit", "strum_macros", "try-traits"]
 
 [dependencies]
 anyhow = "1.0"
+async-stream = { version = "0.3", optional = true }
```

Comment:
You can use https://crates.io/crates/cargo-sort to re-order them alphabetically.